java/ql/test/library-tests/Encryption/secure.ql

```diff
@@ -2,5 +2,5 @@ import default
 import semmle.code.java.security.Encryption
 
 from StringLiteral s
-where s.getLiteral().regexpMatch(getSecureAlgorithmRegex())
+where s.getRepresentedString().regexpMatch(getSecureAlgorithmRegex())
```

Comment:
This test now fails.

Reply:
It looks like the underlying issue was the regex pattern used by `getASecureAlgorithmName()` for AES; it always required `[^a-zA-Z]` after the name, which previously (unintentionally?) matched the `"` of the literal.
I have adjusted the pattern in the latest commit, please let me know if the change is fine.